<!DOCTYPE html>
<html><head><title>Throw</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">17 references to Throw</div><div class="rA">Assembly-CSharp (15)</div><div class="rG" id="Assembly-CSharp"><div class="rF"><div class="rN">Assets\UnitTests\UnitTests.cs (15)</div>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1205"><b>1205</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).Do(x =&gt; list.Add(x), ex =&gt; list.Add(100), () =&gt; list.Add(1000)).Subscribe(_ =&gt; { }, _ =&gt; { }, () =&gt; { });</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1224"><b>1224</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).Do(observer).Subscribe(_ =&gt; { }, _ =&gt; { }, () =&gt; { });</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1244"><b>1244</b>Observable.<i>Throw</i>&lt;int&gt;(new Exception())</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1277"><b>1277</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).DoOnCompleted(() =&gt; list.Add(1000)).Subscribe(_ =&gt; { }, _ =&gt; { }, () =&gt; { });</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1296"><b>1296</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).DoOnError(_ =&gt; list.Add(100)).Subscribe(_ =&gt; { }, _ =&gt; { }, () =&gt; { });</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1319"><b>1319</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception()))</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1340"><b>1340</b>Observable.Range(1, 5).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).DoOnTerminate(() =&gt; list.Add(1000)).Subscribe(_ =&gt; { }, _ =&gt; { }, () =&gt; { });</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1740"><b>1740</b>var xs = Observable.Return(2, Scheduler.ThreadPool).Concat(Observable.<i>Throw</i>&lt;int&gt;(new InvalidOperationException()))</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1759"><b>1759</b>Observable.Return(2).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())),</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1760"><b>1760</b>Observable.Return(99).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())),</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1776"><b>1776</b>Observable.Return(2).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())),</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1777"><b>1777</b>Observable.Return(99).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception()))</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#4367"><b>4367</b>Observable.<i>Throw</i>&lt;string&gt;(ex).Materialize().ToArray().Wait().IsCollection(Notification.CreateOnError&lt;string&gt;(ex));</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#6688"><b>6688</b>Observable.Range(1, 3).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).Materialize()</a>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#6819"><b>6819</b>m = Observable.Range(1, 3).Concat(Observable.<i>Throw</i>&lt;int&gt;(new Exception())).Materialize().ToArrayWait();</a>
</div>
</div>
<div class="rA">Assembly-CSharp-firstpass (2)</div><div class="rG" id="Assembly-CSharp-firstpass"><div class="rF"><div class="rN">Assets\Plugins\UniRx\Scripts\Observable.ErrorHandling.cs (1)</div>
<a href="../Assets/Plugins/UniRx/Scripts/Observable.ErrorHandling.cs.html#126"><b>126</b>: Observable.<i>Throw</i>&lt;TSource&gt;(ex);</a>
</div>
<div class="rF"><div class="rN">Assets\Plugins\UniRx\Scripts\Operators\Defer.cs (1)</div>
<a href="../Assets/Plugins/UniRx/Scripts/Operators/Defer.cs.html#26"><b>26</b>source = Observable.<i>Throw</i>&lt;T&gt;(ex);</a>
</div>
</div>
</body></html>